modules/core/src/main/java/org/apache/ignite/internal/processors/platform/utils/PlatformConfigurationUtils.java

```diff
@@ -289,7 +289,7 @@ public static CacheConfiguration readCacheConfiguration(BinaryRawReaderEx in, Cl
      * @param in Reader.
      * @return Expiry policy factory.
      */
-    private static Factory<? extends ExpiryPolicy> readExpiryPolicyFactory(BinaryRawReader in) {
+    public static Factory<? extends ExpiryPolicy> readExpiryPolicyFactory(BinaryRawReader in) {
```

Comment:
This change and the one and below seem to be unnecessary (no usages outside of this class)

Reply:
I've made this method public in order to reuse it in ClientCacheConfigurationSerializer.java
when we need to parse a new client config section:
```
     case EXPIRY_POLICY: 
                 .....
```
So, I think it's ok